frontend/www/js/omegaup/components/problem/Details.vue

```diff
@@ -351,6 +352,7 @@ export default class ProblemDetails extends Vue {
   @Prop({ default: null }) problemAlias!: null | string;
   @Prop() isAdmin!: boolean;
   @Prop({ default: false }) showVisibilityIndicators!: boolean;
+  @Prop({ default: () => new Date() }) nextSubmissionTimestamp!: Date;
```

Comment:
```suggestion
  @Prop({ default: () => new Date() }) nextSubmissionTimestamp!: null | Date;
```

los tipos de TypeScript indican que esto puede ser nulable, así que hay que declararlo como tal por consistencia. _pero_ ahora se le está pasando esto como nulable a `<omegaup-arena-runsubmit-popup>`, que _no_ espera que pueda ser nulo, así que podría arrojar excepciones! entonces hay que modificar frontend/www/js/omegaup/components/arena/RunSubmitPopup.vue para que ahí también pueda ser nulo, y corregir el código para los casos en los que es nulo.